rules.neon

```diff
@@ -2,8 +2,6 @@ rules:
 	- mglaman\PHPStanDrupal\Rules\Drupal\Coder\DiscouragedFunctionsRule
 	- mglaman\PHPStanDrupal\Rules\Drupal\GlobalDrupalDependencyInjectionRule
 	- mglaman\PHPStanDrupal\Rules\Drupal\PluginManager\PluginManagerSetsCacheBackendRule
-	- mglaman\PHPStanDrupal\Rules\Deprecations\AccessDeprecatedConstant
```

Comment:
Do we need this rule in case future constants are deprecated? Or does PHP now support reading comments from constants... I think it does, so this rule isn't needed. Worth double checking.

Reply:
Yeah it's been out for a while https://x.com/markusstaab/status/1765984217937154473